site/docs/community.md

```diff
@@ -100,3 +102,56 @@ Recruitment of community members should not be conducted through direct messages
+
+### What are the responsibilities of a committer?
+
+In the Iceberg project, committers are community members that can review and commit changes to Iceberg repositories. Reviewing is the primary responsibility of committers.
+
+### How are new committers added?
+
+Starting from the foundation guidelines, committers are nominated and discussed by the PMC, which uses a consensus vote to confirm a new committer. This vote is the only formal requirement in the Iceberg community — there are no other requirements, such as a minimum period of time or a minimum number of contributions. Similarly, there is no length of time or number of commits that automatically qualify someone to be a committer.
+
+Committers are added when someone has built trust with PMC members that they have good judgment and are a reliable reviewer.
+
+### What does the PMC look for?
+
+PMC members typically look for candidates to have demonstrated a few qualities:
+
+* **Conduct** — Committers are representatives of the project and are expected to follow the [ASF Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
+* **Judgment** — Committers should know the areas where they are qualified to evaluate a change and when to bring in other opinions.
+* **Quality** — Personal contributions are a strong signal. Contributions that don’t require major help demonstrate the context and understanding needed to reliably review changes from others. If a contributor often needs guidance, they are probably not ready to guide others.
+* **Consistency** — Reviewing is the primary responsibility of a committer. A committer should demonstrate they will consistently apply their context and understanding to help contributors get changes in and ensure those changes are high quality.
+
+### How do I demonstrate those qualities?
+
+To be a committer, a candidate should act like a committer so that PMC members can evaluate the qualities above. PMC members will ask questions like these:
+
+* Has the candidate been a good representative of the project in mailing lists, Slack, github, and other discussion forums?
+* Has the candidate followed the ASF Code of Conduct when working with others?
+* Has the candidate made independent material contributions to the community that show expertise?
+* Have the candidate’s contributions been stable and maintainable?
+* Has the candidate’s work required extensive review or significant refactoring due to misunderstandings of the project’s objectives?
+* Does the candidate apply the standards and conventions of the project by following existing patterns and using already included libraries?
+* Has the candidate participated in design discussions for new features?
+* Has the candidate asked for help when reviewing changes outside their area of expertise?
+* How diverse are the contributors that the candidate reviewed?
```

Comment:
Just for my understanding: What do we mean by "diverse contributors" here? Diverse by the company they work for or diverse by the area of code they contribute to or something else?

Reply:
My take about "diverse" here is in the number of different contributors reviewed by the candidate (not affiliation).